tutor/src/components/notes/sections-filter.jsx

```diff
@@ -26,13 +26,25 @@ class SectionsFilter extends React.Component {
     }
   }
 
+  @action.bound onSelectAll() {
+    this.props.selected.clear();
+    this.choices.map(c => {
+      this.props.selected.push(c);
+    });
+  }
+
+  @action.bound onSelectNone() {
+    this.props.selected.clear();
+  }
+
   @computed get choices() {
     return this.props.notes.summary.sorted().map((s) => (
       {
         id: s.id,
         title: (
           <span>
-            <ChapterSection chapterSection={s.chapter_section} />
+            <ChapterSection chapterSection={s.chapter_section} hideClassName={true} />
+            &nbsp;
             <span>{s.title}</span>
```

Comment:
Rather than inserting a raw space it's better to add a margin.  You can wrap any component that accepts a className prop with styled components like this:
```js
const FilteredChapterSection = styled(ChapterSection)`
  margin-right: 1rem;
  font-weight: bold;
`
```
And then you could render: `< FilteredChapterSection chapterSection={s.chapter_section} />` here